media/js/mozorg/contact-spaces.js

```diff
@@ -478,8 +490,6 @@
                 mozMap.hideMapLegend();
                 // reposition markers above the labels
                 mozMap.setLabelLayerIndex(1);
-
-                mozMap.updateTabState('contact');
```

Comment:
Not introduced in this change, but here if `state === 'contact'` is true, we're calling `mozMap.unbindCommunityNav();` to unbind the communities navigation, but we're seemingly not not also calling `mozMap.unbindSpacesNav();`